more_itertools/more.py

```diff
@@ -253,24 +253,28 @@ def next(self):
         return self.__next__()
 
     def _get_slice(self, index):
-        start = index.start
-        stop = index.stop
-
-        if (
-            ((start is not None) and (start < 0)) or
-            ((stop is not None) and (stop < 0))
-        ):
-            stop = None
-        elif (
-            (start is not None) and (stop is not None) and (start > stop)
-        ):
-            stop = start + 1
+        # Normalize the slice's arguments
```

Comment:
This procedure is the same as what [CPython does](https://github.com/python/cpython/blob/2ddf5a19c3a06978edff2c8ba0aaf5df3528204a/Objects/sliceobject.c#L221-L233) for normal sequences.